src/open_inwoner/react/lib/web-component/loader.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  createContextsForComponent,
+  findWebComponentsOnPage,
+  runAfterLoadHooks,
+  runBeforeLoadHooks,
+  runErrorHooks,
+  wcRegistry,
+  WebComponentKey,
+} from '.';
+
+/**
+ * Load a single web component with plugin lifecycle
+ */
+export const wcLoader = async (name: WebComponentKey): Promise<void> => {
+  // Skip if already defined
+  if (customElements.get(name)) return;
```

Comment:
Some debug tooling couldn't hurt:

```suggestion
  if (customElements.get(name)) {
      console.debug(`Custom element ${name} is already defined, aborting loading for this element`);
  }
```